embedchain/app.py

```diff
@@ -123,6 +129,15 @@ def __init__(
         if self.cache_config is not None:
             self._init_cache()
 
+        # If memory_config is provided, initializing the memory ...
+        self.mem0_client = None
+        if self.memory_config is not None:
+            try:
+                from mem0 import Mem0
+            except ImportError:
+                raise ImportError("Please install Mem0 with `pip install memzero`") from None
```

Comment:
Let's make memzero as default dependency of embedchain since it doesn't have any extra dependencies.

Reply:
Okay, will do it.